scripts/release/discord-release-message.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @file
+ * Send release message to Discord #releases channel.
+ */
+import process from 'node:process';
+
+const releaseVersion = process.argv[2];
+const discordReleasesRoleId = process.env.DISCORD_RELEASES_ROLE_ID;
+const discordReleasesWebhookUrl = process.env.DISCORD_RELEASES_WEBHOOK_URL;
+const githubReleaseUrl = `https://github.com/simple-icons/simple-icons/releases/tag/${releaseVersion}`;
+
+try {
+  await globalThis.fetch(discordReleasesWebhookUrl, {
+    method: 'post',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      content: `<@&${discordReleasesRoleId}> ${githubReleaseUrl}`,
+    }),
+  });
+} catch {
+  console.error('Failed to send release message to Discord.');
```

Comment:
```suggestion
  console.error('Failed to send release message to Discord.');
  process.exit(1);
```

Reply:
I intentionally did not exit with code 1 because I don't want to block the rest of the steps.

Reply:
Use [`continue-on-error`](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error) on the GHA step.